src/Sylius/Bundle/CoreBundle/Fixture/Factory/OrderExampleFactory.php

```diff
@@ -318,4 +327,34 @@ protected function setOrderCompletedDate(OrderInterface $order, \DateTimeInterfa
             $order->setCheckoutCompletedAt($date);
         }
     }
+
+    protected function fulfillOrder(OrderInterface $order): void
+    {
+        $this->applyPaymentCompleteStateTransition($order);
+        $this->applyShipmentShipStateTransition($order);
+    }
+
+    protected function applyPaymentCompleteStateTransition(OrderInterface $order): void
+    {
+        /** @var PaymentInterface $payment */
+        $payment = $order->getPayments()->first();
+
+        $stateMachine = $this->stateMachineFactory->get($payment, PaymentTransitions::GRAPH);
+        if ($stateMachine->can(PaymentTransitions::TRANSITION_COMPLETE))
+        {
+            $stateMachine->apply(PaymentTransitions::TRANSITION_COMPLETE);
+        }
+    }
+
+    protected function applyShipmentShipStateTransition(OrderInterface $order): void
+    {
+        /** @var ShipmentInterface $shipment */
+        $shipment = $order->getShipments()->first();
```

Comment:
For more future proof solutions, you should iterate over this collection and complete all of the shipments that can be shipped.